### PR TITLE
Adding support for custom paramter feature

### PR DESCRIPTION
**Pull request by tglstory**
also removed 물론 from nouns.

### CODE REVIEW

**Comment by tglstory** [comment]
This will make it easier for 3rd party developers to customize this analyzer.

File: src/main/scala/com/twitter/penguin/korean/tokenizer/KoreanTokenizer.scala [hunk 1]
```diff
@@ -41,26 +41,41 @@ object KoreanTokenizer {
...
+    WEIGHT_TOKENS: Float = 0.18f,
+    WEIGHT_UNKNOWNS: Float = 0.3f,
+    WEIGHT_WORDS: Float = 0.3f,
+    WEIGHT_UNKNOWN_COVERAGE: Float = 0.5f,
+    WEIGHT_FREQ: Float = 0.2f,
+    WEIGHT_POS_UNKNOWNS: Float = 10.0f,
+    WEIGHT_EXACT_MATCH: Float = 0.5f,
+    WEIGHT_ALL_NOUN: Float = 0.1f,
+    WEIGHT_PREFFERED_PATTERN: Float = 0.6f,
+    WEIGHT_DETERMINER: Float = -0.01f,
```
Comment (nlpenguin) [inline src/main/scala/com/twitter/penguin/korean/tokenizer/KoreanTokenizer.scala]: Thanks for the change! Please switch to lower-camel cases as they are not constants any more.

File: src/main/scala/com/twitter/penguin/korean/tokenizer/KoreanTokenizer.scala [hunk 2]
```diff
@@ -41,26 +41,41 @@ object KoreanTokenizer {
...
+    WEIGHT_EXCLAMATION: Float = 0.01f,
+    // supress suffix when tied
+    WEIGHT_INITIAL_POSTPOSITION: Float = 0.2f,
+    WEIGHT_HA_VERB: Float = 0.3f,
+    PREFERRED_PATTERNS: Seq[Seq[KoreanPos]] = Seq(Seq(Noun, Josa), Seq(ProperNoun, Josa))
```
Comment (nlpenguin) [inline src/main/scala/com/twitter/penguin/korean/tokenizer/KoreanTokenizer.scala]: Thanks for the change! Please switch to lower-camel cases as they are not constants any more. i.e) WEIGHT_TOKENS -> weightTokens

**Comment by nlpenguin** [comment]
Thank you so much for doing this!! I will refactor a bit in a separate branch.

**Pull request merged**